system/jlib/jstats.cpp

```diff
@@ -2108,6 +2169,51 @@ class CRootStatisticCollection : public CStatisticCollection
 };
 
 
+StatsScopeId globalScopeId(SSTglobal, (unsigned)0);
+class GlobalStatisticCollection : public CStatisticCollection
+{
+public:
+    GlobalStatisticCollection(IPropertyTree * root) : CStatisticCollection(nullptr, globalScopeId)
```

Comment:
This code belong in workunit.cpp, rather than jlib, because it is specific to the way stats are represented in workunits.  It should be possible to implement using the public interface for the stats collection.